build/lib/release.sh

```diff
@@ -295,7 +295,7 @@ function kube::release::build_hyperkube_image() {
   local -r version="$3"
   local -r save_dir="${4-}"
   kube::log::status "Building hyperkube image for arch: ${arch}"
-  ARCH="${arch}" REGISTRY="${registry}" VERSION="${version}" \
```

Comment:
Why do we remove this?

Reply:
initially i thought REGISTRY can be passed through automatically, not the case apparently.